server/lib/routes/token.js

```diff
@@ -25,14 +25,19 @@ async function tokenRoutes(server) {
 
       const token = uniqid()
 
-      await db
-        .collection('tokens')
-        .doc(secretId)
-        .set({
-          token,
-          subscriptionId
-        })
-      reply.send({ token })
+      try {
+        await db
+          .collection('tokens')
+          .doc(secretId)
+          .set({
+            token,
+            subscriptionId
+          })
+        reply.send({ token })
+      } catch (error) {
+        request.log.error(error.message)
```

Comment:
with fastify I think we can log the error directly, can you check what the api for that looks like?

Reply:
@simoneb Do you mean allowing the default fastify error handler to handle this? I guess I can get rid of the try-catch since fastify automatically logs the error and sends a 500 reponse.

Reply:
that's also an option, yes, in fact I think we may need to do a try/catch here at all!